externs/olx.js

```diff
@@ -25,6 +25,14 @@ olx.AttributionOptions.prototype.html;
 
 
 /**
+ * Tile ranges.
+ * @type {Object.<string, Array.<ol.TileRange>>|undefined}
+ * @api
+ */
+olx.AttributionOptions.prototype.tileRanges;
```

Comment:
No, we were discussing getting rid of this and changing the way attributions work. Should not be `@api`.